LibreNMS/Polling/ConnectivityHelper.php

```diff
@@ -201,8 +201,9 @@ private function updateAvailability(bool $previous, bool $status): void
     private function savePingStats(FpingResponse $ping_response): void
     {
         $perf = $ping_response->toModel();
+        $perf->debug = ['poller_name'=>Config::get('distributed_poller_name')];
         if (! $ping_response->success() && Config::get('debug.run_trace', false)) {
-            $perf->debug = $this->traceroute();
+            $perf->debug = array_merge($perf->debug, $this->traceroute());
```

Comment:
```suggestion
            $perf->debug = ['poller_name' => Config::get('distributed_poller_name'), $perf->debug, ...$this->traceroute());
```

Could do it like this too.